packages/web/src/routes/docs/contributors/local-stats/+page.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Local Statistics
+---
+
+Harper keeps track of certain aspects of your writing.
+Things like:
+
+- What words to misspell most often?
```

Comment:
This should probably be something like `What words are misspelled most often` or `What words do you misspell most often`